cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java

```diff
@@ -419,12 +429,10 @@ private File buildDependencyJar(File targetDir) throws IOException {
       @Override
       public boolean accept(String className, URL classUrl, URL classPathUrl) {
         // Exclude the spark-assembly and scala
-        if (className.startsWith("org.apache.spark")
+        return !(className.startsWith("org.apache.spark")
```

Comment:
I was intended not to simplify in the way that IntelliJ suggested as it is clear that we want to exclude everything from Spark and also everything in Hadoop. Can you undo this change?